misc/acrn-config/launch_config/README

```diff
@@ -1 +1,8 @@
-TODO: parse board_info XML, scenario XML and devicemodel param XML to generate launch script for post-launched vm under devicesmodel/samples/$(BOARD)/ folder
+Please run launch_cfg_gen.py to generate board related configuration patch, the patch would be applied on current acrn-hypervisor git tree automatically.
+
+usage: python3 launch_cfg_gen.py [h] --board <board_info_file> --scenario <scenario info file> --launch <launch info file>
+positional arguments:
+  board_info_file  : file name of the board info XML
+  scenario_info_file  : file name of the scenario info XML
+  launch_info_file :  file name of the launch info
+  uosid :  the vm id which to launch:[0..max uosid id]
```

Comment:
How does uosid used? the usage doesn't show the use of uosid.

Reply:
i will update more specified usage for the uosid option.